src/MongoDB.Driver/MongoDB.Driver.csproj

```
@@ -16,16 +16,31 @@
   <ItemGroup>
     <Compile Include="..\MongoDB.Shared\DictionaryComparer.cs" Link="Shared\DictionaryComparer.cs" />
     <Compile Include="..\MongoDB.Shared\SequenceComparer.cs" Link="Shared\SequenceComparer.cs" />
+    <Compile Include="..\MongoDB.Shared\Hasher.cs" Link="Shared\Hasher.cs" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.100.14" />
```

can you support older version as i suggested here ?
we cant upgrade now our driver because of this dep

https://github.com/mongodb/mongo-csharp-driver/pull/1401/files

```suggestion
    <PackageReference Include="AWSSDK.SecurityToken" Version="[3.7.0,)" />
```

---

This PR is actually part of the v3 iteration development, so the changes will be available together with separating AWS dependencies into the new package. We will consider to take your proposed AWS dependency as a part of next v2 iteration (which was not planned yet).

---

why not to include this little change ? 
v3 probably have breaking changes (?)
this fix will make me to update 2.x without worries and without effort

---

Hi @barnuri, 
As @sanych-sun mentioned, this PR is part of the next 3.x version, while 2.x would is maintained in 2.x branch.

If you think this change should be a part of 2.x and/or 3.x, could you please submit a Jira ticket explaining the issue? Then we could track it independently from this PR.

---

Just realized it's already tracked in [CSHARP-5209](https://jira.mongodb.org/browse/CSHARP-5209)
We'll consider this change for 3.x as part of [CSHARP-4911](https://jira.mongodb.org/browse/CSHARP-4911).

---

yes, there is tickets and there is a PR of one line change
why to wait for end of September for the big change of 3.x when you can easily solve the issue by supporting old versions
https://github.com/mongodb/mongo-csharp-driver/pull/1401